lib/heroku_rails_deploy/deployer.rb

```diff
@@ -0,0 +1,103 @@
+require 'optparse'
+require 'yaml'
+
+module HerokuRailsDeploy
+  class Deployer
+    attr_reader :config_file, :args
+
+    class Options < Struct.new(:environment, :revision)
```

Comment:
I'm not crazy about that rule. For this case we'd have to use the block form of `Struct.new` i.e.

```
Options = Struct.new(:environment, :revision) do 
  def self.create_default(app_registry)
    new(app_registry.keys.first, 'HEAD')
  end
end
```
I'm not crazy about this since it uses an unnecessary closure and is different than most class declarations that developers are used to. I don't find the argument about the superfluous class level very compelling since `Object` already has 13 ancestors in dandelion!